pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1314,32 +1283,11 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
                       console.error('Failed to run rancher desktop networking host-switch.exe process:', error);
                     }
                   } else {
-                    await this.writeFile('/etc/init.d/host-resolver', SERVICE_SCRIPT_HOST_RESOLVER, 0o755);
                     await this.writeFile('/etc/init.d/dnsmasq-generate', SERVICE_SCRIPT_DNSMASQ_GENERATE, 0o755);
-                    // As `rc-update del …` fails if the service is already not in the run level, we add
-                    // both `host-resolver` and `dnsmasq` to `default` and then delete the one we
-                    // don't actually want to ensure that the appropriate one will be active.
```

Comment:
This is still valid until we delete `dnsmasq` and `dnsmasq-generate`.

Alternatively, delete the file directly from `/etc/runlevels/default`.